Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/RuntimeOutputLog/OutputsLogHeader.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import formatMessage from 'format-message';
+
+import { DebugPanelTabHeaderProps } from '../types';
+
+export const RuntimeOutputLogHeader: React.FC<DebugPanelTabHeaderProps> = () => {
+  return (
+    <div
+      css={css`
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+      `}
```

Comment:
In some of the components you've added I see 3 different ways of adding styles:

1. passing an object to the `css` JSX property
2. passing an `emotion` template directly to the `css` JSX property
3. storing an `emotion` template in a variable and that passing that variable to the `css` JSX property

I'd recommend sticking to one convention

Reply:
So, option 1 is used when there is no reusability. We need to use Option 2/3 when multiple div's are reusing the class. It makes sense to stick to option 1 and option 2 thereby removing option 3.

Reply:
The case u marked definitely needs changing